### PR TITLE
Rename catalogue_item_properites to properties #661

### LINKED ISSUE #661: Rename catalogue_item_properites to properties catalogue categories
Corresponds to https://github.com/ral-facilities/inventory-management-system-api/issues/293.

### PR DESCRIPTION
**Pull request by joshuadkitenge**
## Description

see #661

The name of the interfaces will be changed in the react hook form prs 
## Testing instructions

Add a set up instructions describing how the reviewer should test the code

- [ ] Review code
- [ ] Check Actions build
- [ ] Review changes to test coverage
- [ ] {more steps here}

## Agile board tracking

closes #661

### CODE REVIEW

**Comment by joelvdavies** [comment]
Can this be pointed at https://github.com/ral-facilities/inventory-management-system-api/pull/294 for the e2e tests please.

File: .github/workflows/ci-build.yml
Comment (joshuadkitenge) [inline .github/workflows/ci-build.yml:120]:
```suggestion
          harbor.stfc.ac.uk/inventory-management-system/ims-api:pr-294
```

File: .github/workflows/ci-build.yml
Comment (joelvdavies) [inline .github/workflows/ci-build.yml:120]: Note to self: Revert before merging.

**Review by joelvdavies — DISMISSED** [review]
LGTM - Don't merge until we have it running on dev.

File: .github/workflows/ci-build.yml
Comment (joelvdavies) [inline .github/workflows/ci-build.yml:120]:
```suggestion
          harbor.stfc.ac.uk/inventory-management-system/ims-api:develop
```

**Review by joelvdavies — APPROVED** [review]

**Pull request merged**
